src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -67,6 +69,23 @@ protected Logger getLogger() {
         return LOGGER;
     }
 
+    /**
+     * Extract a simple URL, throwing out extra path/query/etc elements.
+     *
+     * @param aLongURL The source URL to be stripped dowwn
+     */
+    private String getSimpleURL(final String aLongURL) {
+        try {
+            final URL url = new URL(aLongURL);
+            final StringBuffer buffer = new StringBuffer().append(url.getProtocol()).append("://")
+                .append(url.getHost()).append("/");
+            return buffer.toString();
+        } catch (MalformedURLException details) {
+            LOGGER.error(details,LOGGER.getMessage(MessageCodes.BUCKETEER_509, aLongURL, details.getMessage()));
+            return "";
```

Comment:
Would it be better to fall back to the `aLongURL` here? That way they'd at least see what they do now, instead of nothing.

Reply:
Yes, it would be better... my mind is still fried by the heat wave.